package.json

```diff
@@ -91,6 +91,18 @@
     "update:genesis": "node deployment/1_createGenesis.js && node deployment/1_createGenesis.js --test --input ../docker/scripts/deploy_parameters_docker.json --out ../docker/scripts/genesis_docker.json",
     "coverage": "npx hardhat coverage",
     "gas:report": "REPORT_GAS=true npx hardhat test",
-    "gas:report:file": "rm -f .openzeppelin/unknown-31337.json && REPORT_GAS=true REPORT_GAS_FILE=true npx hardhat test"
+    "gas:report:file": "rm -f .openzeppelin/unknown-31337.json && REPORT_GAS=true REPORT_GAS_FILE=true npx hardhat test",
+
+    "saveDeployment:mumbai": "mkdir -p deployments/mumbai_$(date +%s) && cp -r deployment/deploy_*.json deployments/mumbai_$(date +%s) && cp .openzeppelin/polygon-mumbai.json deployments/mumbai_$(date +%s) && cp deployment/genesis.json deployments/mumbai_$(date +%s) && mkdir -p deploymentOutput && cp deployment/deploy_*.json deploymentOutput",
```

Comment:
let's move this to a bash script and make an `exec:script` command if we have the time, just so we can handle all save/deploy commands for all networks in one place as needed there and not pollute npm command list overtime with more complicated inline commands (ok if we just add a TODO comment for now)

Reply:
https://linear.app/snapchain/issue/SNA-370/extract-long-inline-npm-script-to-sh